eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantFilterValues.java

```diff
@@ -0,0 +1,85 @@
+package uk.ac.ebi.eva.server.ws;
+
+import uk.ac.ebi.eva.lib.repository.VariantEntityRepository;
+import uk.ac.ebi.eva.server.Utils;
+
+class VariantFilterValues {
```

Comment:
The setter methods in this class should be private, because attribute values are set at creation time and never modified. A filter could be set atomically, as in:

```java
private void setMafFilter(String relation, Double value) {
    setMafOperator(Utils.getRelationalOperatorFromRelation(relation));
    setMafvalue(Utils.getValueFromRelation(value));
}
```

This small change could be beneficial for the refactoring to a generic solution for any filter.